test/plugins/GenerateKeras/GenerateKeras.spec.js

```diff
@@ -16,6 +16,7 @@ describe(pluginName, function () {
     const projectName = 'testProject';
     const manager = new PluginCliManager(null, logger, gmeConfig);
     const ARCHITECTURE = testFixture.ARCHITECTURE;
+    const runPythonCode = testFixture.executePythonCode;
```

Comment:
This is a pretty minor point but it is odd to me that we are aliasing the method to another name in the PR that introduces it. Is `runPythonCode` a better name than `executePythonCode`? Basically, I am a little concerned when a method is named and then renamed in the same PR... It suggests that the name might not be the best name for it.

To be clear, I am not sure if that is the case here (both seem fine to me). It may just be the case that we should use `executePythonCode` instead here but I wanted to check if there was a reason we were renaming it since that might be a problem/annoyance elsewhere, too. For example, if this was just because `runPythonCode` is shorter and more ergonomic, it might be worth renaming to `runPythonCode` since it could be annoying in the future.

Reply:
Agreed, I should have stuck to a single name `runPythonCode`.